kola-denylist.yaml

```
@@ -7,7 +7,7 @@
   tracker: https://github.com/coreos/coreos-assembler/pull/1478
 - pattern: ext.config.kdump.crash
   tracker: https://github.com/coreos/fedora-coreos-tracker/issues/1430
-  snooze: 2023-09-28
+  snooze: 2023-10-13
```

```suggestion
  snooze: 2023-10-24
```

Let's just move this until the F39 final target release date. Then all streams will have moved to F39 and this will no longer be an issue.